recipes/vowpalwabbit/meta.yaml

```diff
@@ -3,13 +3,13 @@ package:
   version: 8.7.0
 
 source:
-  url: https://scgrahamarchive.blob.core.windows.net/vowpal-wabbit/vowpal_wabbit_2ee2bcf_rapidjson.tgz 
-  sha256: befed5ecd42decd533b8e11a98f0e76f9d48951c085b52786277d86ec06923ac
+  url: https://files.pythonhosted.org/packages/2f/fb/1dc86a73579f892762aabebf6fca3c492f77a1f5a6d1035e6e6b6ead89e6/vowpalwabbit-8.7.0.tar.gz
```

Comment:
Can you look at the example and change this with the predicable URL so the update bot can work?

Reply:
fixed